src/components/HomePage/Banner.js

```diff
@@ -24,6 +24,7 @@ import {
 
 // These dates are displayed in the user's timezone
 const monthFormatter = new Intl.DateTimeFormat('en-US', { month: 'short' });
+const dayOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```

Comment:
Move this logic into datetime_utils by creating a function called "getDayOfWeek"